packages/waas/src/intents/base.ts

```diff
@@ -18,6 +18,10 @@ function isSessionStorageAvailable() {
   return typeof window === 'object' && typeof window.sessionStorage === 'object'
 }
 
+export function getLocalTime() {
+  return new Date().getTime()
```

Comment:
interesting.. `new Date().getTime()` works differently then `Date.now()` ?

Reply:
I've confirmed, `Date.now()` is very weird, and does not consider day light savings